<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaviFileSystem</class>
 <widget class="QWidget" name="NaviFileSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>353</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolRefresh">
         <property name="toolTip">
          <string>Refresh file system view</string>
         </property>
         <property name="statusTip">
          <string>Refresh file system view</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolShowAll">
         <property name="toolTip">
          <string>Show / Hide all files</string>
         </property>
         <property name="statusTip">
          <string>Show / Hide all files</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="user-bookmarks"/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolCollapseAll">
         <property name="toolTip">
          <string>Collapse all folders</string>
         </property>
         <property name="statusTip">
          <string>Collapse all folders</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::ListRemove"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolNaviRoot">
         <property name="toolTip">
          <string>Show computer file system</string>
         </property>
         <property name="statusTip">
          <string>Show computer file system</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="computer"/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolOpenSelected">
         <property name="toolTip">
          <string>Open selected file</string>
         </property>
         <property name="statusTip">
          <string>Open selected file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolNewFolder">
         <property name="toolTip">
          <string>Add new folder</string>
         </property>
         <property name="whatsThis">
          <string>Add new folder</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="folder-new"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolNewFile">
         <property name="toolTip">
          <string>Add new file</string>
         </property>
         <property name="statusTip">
          <string>Add new file</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="document-new"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolEditSelected">
         <property name="toolTip">
          <string>Rename selected file / folder</string>
         </property>
         <property name="statusTip">
          <string>Rename selected file / folder</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="mail-message-new"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolDeleteSelected">
         <property name="toolTip">
          <string>Delete selected file / folder</string>
         </property>
         <property name="statusTip">
          <string>Delete selected file / folder</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="user-trash"/>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
